lib/filewatch/sincedb_collection.rb

```diff
@@ -68,35 +68,32 @@ def associate(watched_file)
         # and due to the window handling of many files
         # this file may not be opened in this session.
         # a new value will be added when the file is opened
-        logger.trace("associate: unmatched")
+        logger.trace("associate: unmatched", :filename => watched_file.filename)
```

Comment:
Is there a reason why not guarded by `logger.trace? &&`

Reply:
```suggestion
        logger.trace? && logger.trace("associate: unmatched", :filename => watched_file.filename)
```